src/libecalc/fixtures/cases/consumer_system_v2/data/consumer_system_v2.yaml

```diff
@@ -85,15 +85,75 @@ INSTALLATIONS:
                   TO: pump1
                 - FROM: pump3
                   TO: pump1
-            OPERATIONAL_SETTINGS:
-              - RATES: [ 4000000, 5000000, 6000000 ]
-                INLET_PRESSURE: 50
-                OUTLET_PRESSURE: 250
-                FLUID_DENSITY: 2
-              - RATES: [ 2000000, 2500000, 3000000 ]
-                INLET_PRESSURE: 50
-                OUTLET_PRESSURE: 125
-                FLUID_DENSITY: 2
+            STREAM_CONDITIONS_PRIORITIES:
+              pri1:
+                pump1:
+                  inlet:
+                    RATE:
+                      VALUE: 4000000
```

Comment:
```
                    RATE:
                      VALUE: 4000000
                      UNIT: SM3/d
```

Make room for units, or condition? Condition might make sense one level up? Condition means "empty stream"?